SQL/database_changelog.txt

```diff
@@ -1,3 +1,15 @@
+8th April 2017, by Shadowlight213
+
+Modified table 'player' adding the column 'exp'
+
+ALTER TABLE player ADD exp mediumtext NOT NULL DEFAULT '';
```

Comment:
if you ask me a structure of
ckey | jobtype | time

would be the most flexible as it allows for any number of jobtype groups